config/galaxy.ini.sample

```diff
@@ -212,6 +212,11 @@ paste.app_factory = galaxy.web.buildapp:app_factory
 # Set to True to instruct Galaxy to install Conda from the web automatically
 # if it cannot find a local copy and conda_exec is not configured.
 #conda_auto_init = False
+# You can set this to True if conda_prefix and job_working_directory are not on the same
```

Comment:
I was thinking: can Galaxy reliably detect if `conda_prefix` and `job_working_directory` are on different volumes and set the option accordingly?